src/parser/jobs/rdm/modules/Gauge.tsx

```diff
@@ -0,0 +1,307 @@
+			this._manaStatistics.white.invulnLoss = whiteModified
+			this._manaStatistics.black.invulnLoss = blackModified
+			return
+		}
+
+		this._whiteManaGauge.modify(whiteModified)
+		this._blackManaGauge.modify(blackModified)
+		//Statistics Gathering
+		if (penalized.white) {
+			this._manaStatistics.white.imbalanceLoss = whiteModified
+		}
+		if (penalized.black) {
+			this._manaStatistics.black.imbalanceLoss = blackModified
+		}
+
+	}
+
+	private onManafication() {
+		let whiteModifier = this.getWhiteMana() * this._manaficationMultiplier
+		let blackModifier = this.getBlackMana() * this._manaficationMultiplier
+
+		//Now calculate and store overcap if any.  This way we can still utilize the Overcap
+		//From core, but track this loss separately.
+		//In EW we will still track this separately even though it will no longer be a multiplier but flat 50|50
+		if (whiteModifier > this._manaCap) 		{
+			this._manaStatistics.white.manaficationLoss = whiteModifier - this._manaCap
+			whiteModifier = this._manaCap
+		}
+
+		if (blackModifier > this._manaCap) {
+			this._manaStatistics.black.manaficationLoss = blackModifier - this._manaCap
+			blackModifier = this._manaCap
+		}
+
+		this._whiteManaGauge.set(whiteModifier)
+		this._blackManaGauge.set(blackModifier)
+	}
+
+	private onGaugeSpender(event: Events['action']) {
+		const modifier = this._spenderModifiers.get(event.action)
+
+		if (modifier == null) {
+			return
+		}
+		const amount = modifier
+
+		this._whiteManaGauge.modify(amount.white)
+		this._blackManaGauge.modify(amount.black)
+	}
+
+	//Returns which Mana should be penalized, white, black, or neither
+	private isOutOfBalance() : {white: boolean, black: boolean} {
+		const whiteMana = this.getWhiteMana()
+		const blackMana = this.getBlackMana()
+
+		if (whiteMana && (blackMana - whiteMana > this._manaDifferenceThreshold)) {
+			//If we have more than 30 Black Mana over White, our White gains are halved.
+			return {white: true, black: false}
+		}
+
+		if (blackMana && (whiteMana - blackMana> this._manaDifferenceThreshold)) {
+			//If we have more than 30 White Mana over Black, our Black gains are halved
+			return {white: false, black: true}
+		}
+
+		return {white: false, black: false}
+	}
+
+	private onComplete() {
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.VERHOLY.icon,
+			content: <Fragment>
+				<Trans id="rdm.gauge.suggestions.white-mana-wasted-content">Ensure you don't overcap your White Mana before a combo, overcapping White Mana indicates your balance was off; and you potentially lost out on Enchanted Combo damage.  You should look to execute at 80/80 or as close to it as possible.</Trans>
+			</Fragment>,
+			tiers: this._severityWastedMana,
+			value: this._whiteManaGauge.overCap,
+			why: <Fragment>
+				<Trans id="rdm.gauge.suggestions.white-mana-wasted-why">You lost {this._whiteManaGauge.overCap} White Mana due to capped Gauge resources</Trans>
+			</Fragment>,
+		}))
```

Comment:
I know that this is somewhat a pre-existing condition, but it would probably be better to collapse these suggestions to consider both the aggregate white and black mana losses and output a single suggestion per type (e.g. based on total overcap, make a suggestion and show the two overcaps in the why).  It would help cut down on suggestion bloat from this module and allow other modules to be more relatively important, rather than this module potentially outputting 8 suggestions for what's really only 4 problems.

Reply:
Guidance I got was that they still wanted the individual types of ways to lose mana to appear separate but were fine with white and black collapsing together.  Collapsed them and threw on an icon ternary based on loss.  It was simple so I went ahead and did it.